components/RightSidebar.js

```diff
@@ -1,30 +1,59 @@
-import styles from "../styles/Home.module.css";
+import styles from "../styles/Sidebar.module.css";
+import Link from "next/link";
 
-export default function RightSidebar() {
+export default function RightSidebar({ screenings, movies, movie }) {
+  const screeningsList = screenings.map((screening) => {
+    let title = "";
+    let seatsAvail = 0;
+
+    // Check available seats
+    screening.rows.forEach((row) => {
+      for (const [key, value] of Object.entries(row)) {
+        if (value === "") {
+          seatsAvail++;
+        }
+      }
+    });
+
+    // If single movie
+    if (movie) {
+      title = movie.title;
+    }
+
+    // If all movies
+    if (movies) {
+      movies.forEach((movie) => {
+        if (movie.id === screening.movieid) {
+          title = movie.title;
+        }
+      });
+    }
+    // Create the list of screenings
+    return (
+      <div className={styles["screening-container"]} key={screening.id}>
+        <li>
+          <p className={styles["title"]}>{title}</p>
+          <p>
+            {screening.date} kl. {screening.time}
+          </p>
+          <div className={styles["screening-container-bottom"]}>
+            <p>{seatsAvail} lediga stolar</p>
```

Comment:
This will look weird if there is only a single chair available. "Pluralization" is a very common (and solved) problem when it comes to localization.